riiif.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'rails', '> 3.2.0'
+  spec.add_dependency 'rails', '>= 4.2', '<6'
```

Comment:
would you mind switching to railties?  This will enable us to get the dependency we need without forcing dependencies on things like ActiveRecord.